CHANGELOG.md

```diff
@@ -7,9 +7,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Changed
 * **BREAKING**: Tertiaire button `.btn-link` redesign (#372).
+* **BREAKING**: SVG iconen (#414).
 
 #### Migratie notities:
 
+##### SVG Iconen (#414)
+
+Het voorschrift voor de carousel maakte voorheen gebruik van Bootstrap's glyphicons. Het SVG-icoon element moet een modifier `icon-next` en `icon-prev` krijgen. Dit zijn standaard Bootstrap modifiers.
```

Comment:
carrousel wordt geschreven met 2x `r` ;)

Reply:
Dat neem ik terug, als Engels de voertaal is voor componenten dan is de enkele `r` wel correct: https://taaleidoscoop.nl/blog/carousel/

Reply:
De enige juist manier om carousel te spellen ;-)

We zijn bootstrap gebasseerd, daar is het ook `.caroussel`